ocs_ci/ocs/ui/base_ui.py

```diff
@@ -836,8 +837,24 @@ def take_screenshot(driver):
     time.sleep(0.5)
 
 
-@retry(TimeoutException, tries=3, delay=3, backoff=2)
-@retry(WebDriverException, tries=3, delay=3, backoff=2)
+def garbage_collector_webdriver():
+    """
+    Garbage Collector for webdriver objs
+
+    """
+    collected_objs = gc.get_objects()
+    for obj in collected_objs:
+        if str(type(obj)) == "<class 'selenium.webdriver.chrome.webdriver.WebDriver'>":
+            try:
+                obj.close()
+            except Exception as e:
+                logger.error(e)
+
+
+@retry(TimeoutException, tries=3, delay=3, backoff=2, func=garbage_collector_webdriver)
+@retry(
+    WebDriverException, tries=3, delay=3, backoff=2, func=garbage_collector_webdriver
+)
```

Comment:
So this function `garbage_collector_webdriver` will be re-tried in case TimeoutException or WebDriverException is raised?

Reply:
The `garbage_collector_webdriver` will close the webdriver session in case of failure

Reply:
@OdedViner , can't this be consolidated to single @retry ? https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/retry.py#L13 , so first parameter could be a tuple containing multiple exceptions. WDYT?

Reply:
done